### DISCUSSION TITLE
How come vcpkg isn't integrated in Visual Studio like NuGet is?

### DISCUSSION THREAD

**Discussion by jrmoore** [microsoft/vcpkg#40127] [Ideas]
I saw some C++ projects using NuGet for dependencies, even if not recommended, the way to handle things graphically through Visual Studio is rather convenient, no need to edit JSONs/XMLs/etc. directly, no need to get down to CMake / MSBuild level and package search is also at hand.

How come vcpkg doesn't have the same level of integration? I only came to know of it recently, but I am curious.

![image](https://github.com/user-attachments/assets/c06bd811-1985-499a-9255-051bbeec258a)

Related: [Visual Studio GUI for managing vcpkg dependencies?](https://github.com/microsoft/vcpkg/discussions/35404)

**Comment by AugP** [comment]
We prioritize feature work based on our capacity and the feedback we get from users. If this is something you would like to see more investment in, I recommend upvoting this thread where we are tracking this issue: https://github.com/microsoft/vcpkg/discussions/35404

We look at the most upvoted feature requests when we plan our roadmap.

**Reply by jrmoore** [reply]
Thanks! Just upvoted that thread, I didn't think it was constituting a feature request.

I was just curious, as a newcomer to vcpkg I found it convoluted compared to what I had seen previously. When I was told it had been integrated with Visual Studio, the first thing I thought was it being a GUI similar to that of NuGet, not a property page, so I was wondering why that would be.

There really isn't a demand for things to be simpler or easier to use? The NuGet GUI, for example, seems straightforward enough even if it needed to be extended. Fingers crossed 🤞🏻 that similar integration materializes in one way or another.
